host-bin/edit-chroot

```diff
@@ -335,15 +363,34 @@ for NAME in "$@"; do
                 dest="$dest.gz"
             fi
         fi
+        # If we're writing to a fat32 filesystem, split the file at 4GB chunks
+        if ! df -T "`getmountpoint "$dest"`" | awk '$2~"^v?fat"{exit 1}'; then
+            SPLIT="${SPLIT:-4GB}"
```

Comment:
At that point, I might as well just put in `4294967295`, since that's the actual file size limit.